Rakefile

```diff
@@ -102,30 +106,75 @@ task :test_all do
   GEMS.each do |repo|
     box "Testing #{repo}"
     Dir.chdir(repo) do
+      FileUtils.rm_f(log_file)
       begin
         # From https://github.com/bundler/bundler/issues/1424#issuecomment-2123080
         # Since we nest bundle exec in bundle exec
         Bundler.with_clean_env do
           bundle_install
-          sh "bundle exec rspec"
           sh "bundle exec rubocop"
+          sh "bundle exec rspec --format documentation --format j --out #{log_file}"
         end
       rescue => ex
         puts "[[FAILURE]] with repo '#{repo}' due to\n\n#{ex}\n\n"
         exceptions << "#{repo}: #{ex}"
+        repos_with_exceptions << repo
       end
     end
   end
 
-  if exceptions.count > 0
-    puts "--------------------"
-    puts "#{exceptions.count} failure(s)"
-    puts "--------------------"
-    puts exceptions.join("\n")
-    raise "Tests failed, search test log for [[FAILURE]] to find the outputs"
+  failures = {}
+  example_count = 0
+  duration = 0.0
+
+  GEMS.each do |gem_name|
+    failures[gem_name] = []
+    log = File.join(gem_name, log_file)
+    file_text = File.read(log)
+    log_json = JSON.parse(file_text)
+    results = log_json["examples"]
+    failures[gem_name] += results.select { |r| r["status"] == "failed" }
```

Comment:
Shouldn't this be `!= "success"`? maybe there is something like `timeout`? Or is it really only those 2 values?